components/41-organisms/header/header.config.js

```diff
@@ -53,26 +53,12 @@ module.exports = {
       }
     },
     {
-      name: 'subsite-hero',
```

Comment:
There's an error on the Detail Layout because this variant is removed.

Besides, I think it'd be wise to retain these variants to have an example of what a header with a hero image looks like.

Reply:
Yes, but they were removed from the twig file...
I'll readd them then.